Neos.Flow/Classes/Http/Client/InternalRequestEngine.php

```diff
@@ -49,7 +49,7 @@ class InternalRequestEngine implements RequestEngineInterface
 
     /**
      * @Flow\Inject(lazy = false)
-     * @var Router
+     * @var RouterInterface
      */
     protected $router;
```

Comment:
You didn't adjust the return type hint of the respective getter (https://github.com/Torsten85/flow-development-collection/blob/39e601e0312fe5bc3a8ef0e9f4a937063e3cda4d/Neos.Flow/Classes/Http/Client/InternalRequestEngine.php#L138)

Reply:
Sorry for that, was a bit in a hurry ;)